css/index.css

```diff
@@ -81,6 +81,8 @@ header nav {
     width: 69%;
     justify-content: space-between;
     display: flex;
+    flex-shrink: 2;
```

Comment:
Loved the responsiveness and fluid design of your project. When I was shrinking the screen towards the phone, the nav looked a little asymmetric. Don't know if that was intended or something that was overlooked.